spec/services/alerts_service_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+describe AlertsService do
+  shared_examples 'a threshold that needs to be alerted' do
```

Comment:
shouldn't this be
```suggestion
  shared_examples 'an alert that needs to be triggered' do
```
?